code/datums/wounds/bones.dm

```diff
@@ -475,7 +472,7 @@
 	. += "<div class='ml-3'>"
 
 	if(severity > WOUND_SEVERITY_MODERATE)
-		if(victim.get_biological_state() == BIO_JUST_BONE)
+		if((limb.biological_state & BIO_BONE) && !(limb.biological_state & BIO_FLESH))
```

Comment:
Could be fun to put these in a seperate proc or make them dynamic (for another PR)

Reply:
yes, for now i shall leave this as-is